main.go

```diff
@@ -27,10 +27,10 @@ const (
 // Config ...
 type Config struct {
 	// Gradle Inputs
-	GradleFile    string `env:"gradle_file"`
-	GradleTasks   string `env:"gradle_task,required"`
-	GradlewPath   string `env:"gradlew_path,file"`
```

Comment:
If the input is annotated with `file`, its existence is checked against the current working directory. In this case, `gradle_file` is relative to `build_root_directory` instead.